src/index.js

```diff
@@ -1 +1,88 @@
-// Lets do this!
+import { toRefs, h, computed, markRaw, watch, getCurrentInstance } from 'vue'
+import { useForm, useField } from 'vee-validate';
+
+/**
+ * For a Schema, find the elements in each of the rows and remap the element with the given function
+ * @param {Array} schema
+ * @param {Function} fn
+ *
+ * @returns {Array}
+ */
+export const mapElementsInSchema = (schema, fn) => schema.map(row => row.map(el => fn(el)))
+
+export default function VeeValidatePluginFactory () {
```

Comment:
Yea I thought about this, vee-validate has some configurations that can be configured either here or on vee-validate side, so I had intentions to experiment with them after the initial implementation. If we are not allowing plugins to have configuration objects then users can easily configure them using vee-validate, either way, both are fine to me. 

A couple of examples so you can have context:

```js
import { configure } from 'vee-validate';

configure({
  // ...
});

const SchemaFormWithPlugin = SchemaFormFactory([
  veeValidatePlugin,
]);

// or with config

const SchemaFormWithPlugin = SchemaFormFactory([
  veeValidatePlugin({
    // ... 
  }),
]);
```

I think it is better to avoid config objects as it might be confusing in some cases, so fully agree.